runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -513,8 +513,18 @@ public void requestFocus()
 			OSXUtil.requestFocus();
 		}
 
-		frame.requestFocus();
-		giveClientFocus();
+		if (System.getProperty("os.name").contains("10"))
+		{
+			frame.setAlwaysOnTop(true);
+			frame.requestFocus();
+			giveClientFocus();
+			frame.setAlwaysOnTop(config.gameAlwaysOnTop());
```

Comment:
Why is `frame.setAlwaysOnTop()` called twice in this block?

Reply:
The first call brings the client to the front, and then the second call sets it back to whatever the config was set to. 

In the case of true, the client would always be on top anyway, but in the case of false, when the client is hidden behind another application, the client is called to the top, client is given focus and then the client can be hidden away again.